.env.example

```
@@ -1 +1,7 @@
-LOG_LEVEL=info
\ No newline at end of file
+PORT=
+MONGO_USERNAME=
+MONGO_PASSWORD=
+MONGO_PORT=
+BASE_URL=
+MONGO_CLAUSTER_NAME=
+CONNECTION_STRING=mongodb+srv://${MONGO_USERNAME}:${MONGO_PASSWORD}@cluster0.vhqcy.mongodb.net/?retryWrites=true&w=majority&appName=${MONGO_CLAUSTER_NAME}
```

```suggestion
CONNECTION_STRING=mongodb://${MONGO_USERNAME}:${MONGO_PASSWORD}@mongo:${MONGO_PORT}/${MONGO_CLUSTER_NAME}?authSource=admin
```

Also, why did you delete `LOG_LEVEL` from this file?